Expensify/App
# Show when a user is typing in a room

@marcaaron please review

Show "[User] is typing..." when typing in the RNC Client. If multiple people are typing we will say "Multiple users are typing..." (I've asked about that in the issue [here](https://github.com/Expensify/Expensify/issues/142357#issuecomment-709710286))

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/142357

### Tests
- Have two open sessions (I did this using one browser and one desktop app client, both logged into different accounts)
- Start typing in one of the sessions, make sure that you see `[User] is typing...` in the header of the chat in the other session.
- Do the same from the other session,  make sure that you see `[User] is typing...` in the header of the chat of the previous session.

---

- Find a friend! (or up the timeout to a high number [here](https://github.com/Expensify/ReactNativeChat/blob/ee5dfdbcc91966acc072203d2996df4bec430aa7/src/lib/actions/Report.js#L264) like I did_
- Start a mobile session in addition to the other two.
- Start typing in two of the sessions, make sure that you see `Multiple users are typing...` in the session you're not typing in.


### Screenshots
Desktop/Web:
![Kapture 2020-10-15 at 20 29 22](https://user-images.githubusercontent.com/4741899/96209774-4291f380-0f25-11eb-8c69-bc0c0438fd73.gif)
